TYPO3.Flow/Classes/TYPO3/Flow/Resource/ResourceManager.php

```diff
@@ -115,14 +118,14 @@ public function initialize()
     /**
      * Imports a resource (file) from the given location as a persistent resource.
      *
-     * On a successful import this method returns a Resource object representing the
+     * On a successful import this method returns a PersistentResource object representing the
      * newly imported persistent resource and automatically publishes it to the configured
      * publication target.
      *
-     * @param string | resource $source A URI (can therefore also be a path and filename) or a PHP resource stream(!) pointing to the Resource to import
+     * @param string | PersistentResource $source A URI (can therefore also be a path and filename) or a PHP resource stream(!) pointing to the Resource to import
```

Comment:
I think that is really supposed to be PHP 'resource', not a Flow PersistentResource